### PR TITLE
Anel 1.x bindings successfully tested with OH2 compatibility layer.

### PR DESCRIPTION
**Pull request by paphko**
I just tested the Anel binding from OH 1.x master branch and it works fine on my setup with OH2 compatibility layer.

### CODE REVIEW

File: distribution/openhabhome/conf/services/anel.cfg [hunk 1]
```diff
@@ -0,0 +1,30 @@
+########################## Anel NET-PwrCtrl Binding ###################################
+#
+# <name> must a an identifier that is also used for the item bindings.
+# Example configuration: anel:anel1.host=net-control
+# Example item binding: Switch f1 { anel="anel1:F1" }
+
+# IP or network address (optional but recommended, defaults to 'net-control')
+#anel:<name>.host=anel1
```
Comment (kaikreuzer) [inline distribution/openhabhome/conf/services/anel.cfg]: you need to remove the "anel:" prefixes within this file!

File: distribution/pom.xml [hunk 2]
```diff
@@ -1,368 +1,373 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (kaikreuzer) [inline distribution/pom.xml]: this file shows too many changes - could you try to make the PR to only show the newly added lines? Thanks!

Comment (paphko) [inline distribution/pom.xml]: I tried several times, but no luck. Git always shows the entire contents replaced although I just added 4 lines. This does not occur for other files (I tested it), just this file. I also checked line endings, still no luck. Sorry, but I don't know how to tell git that only 4 lines changed...
Any idea?

**Comment by paphko** [comment]
Ok, I identified the cause: line endings! The distribution pom.xml has different line ending than the other files. And git was configured with **core.autocrlf = true**. After setting it to false, git did not change all line endings and only the lines of interest are marked as changed.

**Comment by kaikreuzer** [comment]
Thanks for fixing the problems!

**Pull request merged**